src/models/objectiveIpdCourse.js

```diff
@@ -0,0 +1,36 @@
+const { Model } = require('sequelize');
+
+export default (sequelize, DataTypes) => {
+  class ObjectiveIpdCourse extends Model {
+    static associate(models) {
+      ObjectiveIpdCourse.belongsTo(models.Objective, { foreignKey: 'objectiveId', onDelete: 'cascade', as: 'objective' });
+      ObjectiveIpdCourse.belongsTo(models.IpdCourse, { foreignKey: 'ipdCourseId', as: 'ipdCourse' });
```

Comment:
You have the belongsTo, but are missing the Hasmany. place them in this file. it helps prevent sequencing issues if all the associations introduced by this table are in this tables model file.